Fake.sln

```diff
@@ -129,6 +129,8 @@ ProjectSection(SolutionItems) = preProject
 	build.fsx = build.fsx
 EndProjectSection
 EndProject
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "Fake.FluentMigrator", "src\app\Fake.FluentMigrator\Fake.FluentMigrator.fsproj", "{F3EEDB98-2413-412D-A52B-8749B2128CA8}"
```

Comment:
You can probably just change the \ to / and you are fine. At least that did it for me last time :)

Reply:
Unfortunately that did not fix it :(

Reply:
The actual error im seeing in the log is: 
error MSB6003: The specified task executable "fsc.exe" could not be run. Permission denied [/home/travis/build/fsharp/FAKE/src/app/Fake.FluentMigrator/Fake.FluentMigrator.fsproj]

Don't know why permissions would be denied, its may be an issue with fsproj config or something?